.rubocop.yml

```diff
@@ -98,5 +98,12 @@ Style/SpecialGlobalVars:
 Style/MissingRespondToMissing:
   Enabled: false
 
-Style/BracesAroundHashParameters:
-  Enabled: false
+Style/HashEachMethods:
+  Enabled: true
+
+Style/HashTransformKeys:
+  Enabled: true
+
+Style/HashTransformValues:
+  Enabled: true
+
```

Comment:
Apparently new configs added that need to be explicitly enabled/disabled. Weird Rubocop.